tools/Generator/Xamarin.Forms.Core.json

```diff
@@ -2407,6 +2491,25 @@
           "inputType": "obj",
           "modelType": "obj",
           "updateCode": "updateSelectedItem"
+        },
+        {
+          "name": "QueryChanged",
+          "defaultValue": "ignore",
+          "modelType": "string * string -> unit",
+          "updateCode": "(fun _ curr (target: Xamarin.Forms.SearchHandler) -> match curr with ValueSome fn -> (target :?> Fabulous.DynamicViews.CustomSearchHandler).QueryChanged <- fn | ValueNone -> ())"
```

Comment:
Those updateCode don't support incremental updates.
Will try to rework that later